composer.json

```diff
@@ -23,7 +23,7 @@
         "symfony/phpunit-bridge": "^3.4|^4.4|^5.0",
         "doctrine/common": "~2.2",
         "doctrine/persistence": "^1.3.3",
-        "doctrine/dbal": "~2.2",
+        "doctrine/dbal": "^2.13",
```

Comment:
Shouldn't this be a dependency instead dev dependency as it's used in the library code?

Reply:
No, you can use the ACL library without Doctrine DBAL. Making DBAL a mandatory dependency would be wrong.

But we should add a conflict rule for DBAL 2.13, to make sure we don't get an earlier release.

Reply:
Ah yes, you're right! I've added the conflict rule!